repos/keg-core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@keg-hub/keg-core",
   "main": "node_modules/expo/AppEntry.js",
-  "version": "6.3.0",
+  "version": "7.0.0",
```

Comment:
technically we should only be updating this version right before a release right

Reply:
Hmm good catch, this must have come from one of my update scripts.
I've reverted it back to 6.3.0